solr/server/solr/configsets/sample_techproducts_configs/conf/solrconfig.xml

```diff
@@ -1597,8 +1601,18 @@
      <valueSourceParser name="myfunc" 
                         class="com.mycompany.MyValueSourceParser" />
     -->
-    
-  
+  <!-- Learning to rank query parser
+
+  http://wiki.apache.org/solr/FIXME
+
+       Query parser used to rerank top docs with a provided  model
+   -->
+       <queryParser enable="${solr.ltr.enabled:false}"
+name="ltr" class="org.apache.solr.ltr.search.LTRQParserPlugin" >
```

Comment:
How about not having the (advanced) parallelisation in the (basic) example here? Though if having then would need to have the equivalent for the transformer's config also for consistency I think.